### DISCUSSION TITLE
my python terminal is not given my a good result like example, usr/local/bin/python3 "/Users/user/py pro/you.py" (base) user@USERs-Air py pro % /usr/local/bin/python3 "/User s/user/py pro/you.py" (base) user@USERs-Air py pro % >>>2+2 zsh: parse error near `>' (base) user@USERs-Air py pro %

### DISCUSSION THREAD

**Discussion by Gabriel29124** [community/community#70186] [Codespaces]
### Select Topic Area

Product Feedback

### Body

usr/local/bin/python3 "/Users/user/py pro/you.py"
(base) user@USERs-Air py pro % /usr/local/bin/python3 "/User
s/user/py pro/you.py"
(base) user@USERs-Air py pro % >>>2+2
zsh: parse error near `>'
(base) user@USERs-Air py pro %

**Comment by Akash1134** [comment]
Hi @Gabriel29124 , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
